src/redux/reducers/index.js

```diff
@@ -71,6 +71,14 @@ const preloadCollections = {
   [defaultCollection.id]: defaultCollection,
 };
 
+const getNextEnabledType = (resourceType, enabledTypes) => enabledTypes
+  .map((type, index, array) => ({
+    type,
+    next: array[(index === array.length - 1) ? 0 : index + 1],
```

Comment:
alternatively, the "next" type could be the 1st on left that is enabled.

...and if it's the 1st enabled, pick the next to the right.